admin-console/src/main/webapp/src/components/SystemBehaviorGroups/DeleteBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,59 @@
+import { ActionGroup, Button, Modal, ModalVariant, Spinner, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { BehaviorGroup } from '../../types/Notifications';
+
+interface DeleteBehaviorGroupModalProps {
+    onDelete: (systemBehaviorGroup?: BehaviorGroup) => Promise<boolean>;
+    bundleName?: string;
+    systemBehaviorGroupName?: string;
+    isOpen: boolean;
+    onClose: () => void;
+
+}
+export const DeleteBehaviorGroupModal: React.FunctionComponent<DeleteBehaviorGroupModalProps> = (props) => {
+    const [ errors, setErrors ] = React.useState(true);
+
+    const onDelete = React.useCallback(async () => {
+        const onDeleteImpl = props.onDelete;
+        const response = await onDeleteImpl();
+        if (response) {
+            props.onClose();
+        } else {
+            alert('Could not delete system behavior group, please try again.');
+        }
+    }, [ props ]);
+
+    const handleDeleteChange = (value: string, event: React.FormEvent<HTMLInputElement>) => {
+        const target = event.target as HTMLInputElement;
+        if (target.value !== props.systemBehaviorGroupName) {
+            return setErrors(true);
+        } else if (target.value === props.systemBehaviorGroupName) {
+            return setErrors(false);
+        }
+    };
+
+    return (
+        <React.Fragment>
+            <Modal variant={ ModalVariant.small } titleIconVariant="warning" isOpen={ props.isOpen }
+                onClose={ props.onClose }
+                title={ `Permanently delete ${ props.systemBehaviorGroupName }` }>
+                { <b>{ props.systemBehaviorGroupName }</b> } {`from ${ props.bundleName ? props.bundleName :
+                    <Spinner /> } will be deleted. You will no longer be able to assign this system behavior
+                    group to event types. `}
```

Comment:
Switching to use the markup instead of the template literal
 ```suggestion
                { <b>{ props.systemBehaviorGroupName }</b> } from { props.bundleName ? props.bundleName :
                    <Spinner /> } will be deleted. You will no longer be able to assign this system behavior
                    group to event types.
```